arangod/RocksDBEngine/RocksDBValue.cpp

```diff
@@ -88,6 +88,22 @@ LocalDocumentId RocksDBValue::documentId(std::string const& s) {
   return documentId(s.data(), s.size());
 }
 
+bool RocksDBValue::revisionId(rocksdb::Slice const& slice, TRI_voc_rid_t& id){
+  if(slice.size() == sizeof(LocalDocumentId::BaseType) + sizeof(TRI_voc_rid_t)){
+    id = rocksutils::uint64FromPersistent(slice.data() + sizeof(LocalDocumentId::BaseType));
+    return true;
+  }
+  return false;
+}
+
+TRI_voc_rid_t RocksDBValue::revisionId(rocksdb::Slice const& slice){
+  TRI_voc_rid_t id;
+  if(revisionId(slice, id)){
+    return id;
+  }
+  THROW_ARANGO_EXCEPTION(TRI_ERROR_INTERNAL);
```

Comment:
please add an error message to this, otherwise we will just have "internal error" without any further details!